app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,13 @@
       </div>
     <%end%>
 
+    <% if info.present? %> 
+    <div class="alert alert-primary notif"> 
+      <button type="button" class="close ml-3" data-dismiss="alert">&times;</button> 
+      <%= info %> 
+    </div> 
```

Comment:
I suggest adding a tab inside this `if` block. Just like you did here: 
![image](https://user-images.githubusercontent.com/21062088/99715844-fc243d00-2a6c-11eb-8b12-28d0e6c90b2b.png)